src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Properties/OutputTypeExValueProviderTests.cs

```diff
@@ -52,8 +52,7 @@ public async Task SetValue(string incomingValue, string expectedOutputTypeValue)
                     SetValues = setValues
                 });
             var provider = new OutputTypeExValueProvider(properties);
-
-            var actualPropertyValue = await provider.OnSetPropertyValueAsync(incomingValue, null);
+            await provider.OnSetPropertyValueAsync(incomingValue, null);
             Assert.Equal(setValues.Single(), expectedOutputTypeValue);
```

Comment:
Looks like it's checking the wrong result, I suspect "actualPropertyValue" is supposed to be in the assert.

Reply:
Looks like `OnSetPropertyValueAsync` has a hardcoded null return, and this test has been broken in this way for a while, so the removal makes sense.